.travis.yml

```diff
@@ -1,13 +1,19 @@
-language: scala
+language: java
```

Comment:
(The other option would be to not include it in the repository but fetch it in `before_install`.)